deployment/deploy.sh

```diff
@@ -14,11 +14,11 @@ kubectl apply -f ./scrumlr-dashboard/configmap.yaml
 kubectl apply -f ./scrumlr-dashboard/secrets.yaml
 
 # scrumlr-database
-kubectl apply -f ./scrumlr-database/deployment.yaml
-kubectl apply -f ./scrumlr-database/service.yaml
-kubectl apply -f ./scrumlr-frontend/deployment.yaml
+
+kubectl apply -f ./scrumlr-database/
```

Comment:
I think Timo explained to my that it's always better to create every single resource instead of creating a 'folder'